### DISCUSSION TITLE
使用PCL启动第三方（像Lunar）的版本？

### DISCUSSION THREAD

**Discussion by Turtle5912** [Meloong-Git/PCL#4473] [提问] [answered]
虽然api问题应该很难解决

**Comment by zkitefly** [comment]
PCL 应该可以启动官方启动器能启动的版本。只不过下载方面不确定要不要做

不过 Lunar 端的文件格式是和官方启动器的格式没有一点相同的，所有我认为如果要支持第三方版本，Lunar 端也不太可能会做适配

**Comment by wuliaodexiaoluo** [comment]
非官方版本不考虑支持

**Comment by WForst-Breeze** [comment, marked as answer]
PCL 应该是不会支持下载安装诸如 Lunar、Badlion 等客户端的，这有一些超出 PCL 的范畴了。毕竟虽然都是 P 字辈的，但总不能把市面上所有 PVP 客户端都塞进 PCL 吧 🤔（同理参阅 #2575）

启动方面只要是和官方格式一样的版本一般都可以启动的，也应该是不会刻意去做对于第三方客户端的启动适配的。

因为既然都用第三方客户端了，那就不如直接去用他们自家的启动器，这样既稳定也不会给 PCL 的开发带来一些用户体验方面收益并不大的开发成本

（个人意见）
